### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Donard97**
**I created this PR to check if our project was as requested.**

_This is a Bookstore project when you can add a book, and remove selected books. In this milestone, I had to style everything._

### Project Requirement:

> - Added style to Bookstore ✔️ 
> - Used information in Zeplin to style the website ✔️  
> - My final design is matching the Zeplin's design ✔️ 

<hr>

_Feel free to review my project and let me know if there are any essential changes 💪 ._

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,41 +1,35 @@
...
-  );
-};
+const Navbar = () => (
+  <nav className="navbar navbar-expand-lg navbar-light bg-light m-4">
+    <div className="container-fluid">
+      <a className="navbar-brand  fw-bold" href="/">
+        <Link className="nav-link" to="/">
+          <h4 className="rangi"> Bookstore CMS </h4>
+        </Link>
+      </a>
```
Comment (henatan99) [inline src/components/Navbar.js]: - [x] I think you don't need to wrap a `Link` element in 'a` tag as they are the same when converted to html. The browser is complaining about it. 

Screenshot from 2021-10-08 12-36-01

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Donard97 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 1 above)
Comment (Donard97) [reply src/components/Navbar.js]: Thank you for letting me know. I changed what you requested me to do.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉 

## Hi @Donard97, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**